assets/helpers/tracking/googleTagManager.js

```diff
@@ -58,5 +58,6 @@ export function init() {
     orderId: getDataValue('orderId', uuidv4),
     currency: getDataValue('currency', getCurrency),
     value: getContributionValue(),
+    paymentMethod: storage.getSession('paymentMethod'),
```

Comment:
If local storage is not available (e.g. private browsing mode), `getSession` will [return null](https://github.com/guardian/support-frontend/blob/master/assets/helpers/storage.js#L59). Do we want this behaviour?

Reply:
I think that's fine, there isn't really anything else we can do.

Reply:
As an alternative, I think if you set `undefined` instead of null that field is not being sent to Google.